### PR TITLE
Count average messages by time intervals + Slovak translation

### LINKED ISSUE #25: Count avg messages by time intervals
Add option to count average messages sent by time period, ex: day, week, month, year

### PR DESCRIPTION
**Pull request by Infinitiv525**
# Description

Added way to calculate the average amount of messages in periods of day, week, month and year
by calculating the age of conversation and then dividing the it by the given periods to find the 
average amount of messages per period.
This averages are displayed in message statistics.
I added all the relevant text to all the languages, however the translations will need to be checked.
I have marked whose with a comment.

I also added a Slovak translation.

I also found a bug, I have marked it with a comment, because I was unable to fix it without disrupting
the functionality too much.

Fixes #25 

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: Main.py [hunk 1]
```diff
@@ -370,7 +371,7 @@ def extract_data(self, conversation):
                     # save call durations, if any
                     call_duration += message.get('call_duration', 0)
                     # fetch conversation creation date
-                    start_date = message['timestamp_ms']
+                    start_date = message['timestamp_ms']  # BUG: doesn't work properly if there are 10 or more JSONs
```
Comment (Kubis10) [inline Main.py:374]: Recently I found this bug too, but it is very strange dependencies

File: langs/Polski.py
Comment (Kubis10) [inline langs/Polski.py:1]: This is good

File: Main.py (hunk 1 above)
Comment (Infinitiv525) [reply Main.py:374]: Should I tak a look at that, or just leave it for later?

Comment (Kubis10) [reply Main.py:374]: You can but in other issue

**Comment by Kubis10** [comment]
I found one bug on group conversation:
image
as you can see, the box is too small, and it cannot be scrolled to view all

**Comment by Kubis10** [comment]
image
now I have 2 scrollbars that are not working

**Comment by Infinitiv525** [comment]
Then maybe a better solution would be to decrease the size of the field of gorup participants. The scrollbars never moved, the bigger one is from the groupchat window that was not added by me. Both windows are scrollable, but just the scrollbar doesn't seem to "move"

**Comment by Kubis10** [comment]
Or on group chat change size of window to be bigger

**Review by Kubis10 — APPROVED** [review]

**Pull request merged**